ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -151,20 +151,16 @@ class _PprofConverter(object):
         self,
         filename: str,
         lineno: int,
-        funcname: typing.Optional[str] = None,
+        funcname: str,
     ) -> pprof_LocationType:
         try:
             return self._locations[(filename, lineno, funcname)]
         except KeyError:
-            if funcname is None:
-                real_funcname = "<unknown function>"
-            else:
-                real_funcname = funcname
```

Comment:
is this related?

Reply:
Yes; we don't want to allow `None` in `FrameType` nor anywhere to be sure we're not going to compare `None` to a `str` at some point when `sort()`ing.

In practice this case does not happen anymore since we always have a funcname since we removed the old memory profiler (based on tracemalloc).